packages/backend-common/src/config.ts

```diff
@@ -24,7 +24,7 @@ export async function loadBackendConfig() {
   /* eslint-disable-next-line no-restricted-syntax */
   const paths = findPaths(__dirname);
   const configs = await loadConfig({
-    env: process.env.NODE_ENV ?? 'development',
+    env: process.env.APP_ENV ?? process.env.NODE_ENV ?? 'development',
```

Comment:
This is one of those places where I feel that good old `||` better conveys what one wants than newfangled `??` :)